### ISSUE TITLE
Steps not accurately shown

### ISSUE THREAD

**Issue by aversluis1** [PerfectFit-project/testing-tickets#82]
I wore the smartwatch yesterday (after not having worn it for a while). 
I ran both the weekly reflection script and the step goal notification (using /EXTERNAL); however, it did not register any steps yesterday. 
Moreover, the step goal of today is much higher than yesterdays goal (9080 vs 7410), which I find strange as no steps were registered.

**Comment by wbaccinelli** [comment]
@bscheltinga can you check this?

**Comment by bscheltinga** [comment]
@wbaccinelli maybe you can pull the data from senseeact to be sure there is no data. Can you share the number of steps per day for the last 14 days with me? I need that, to look into the goal calculation. The step goal calculation shows this behaviour because of missing data, I've seen that before.

@aversluis1 probably, the /EXTERNAL_pa_step_goal_notification probably said that you need to contact the researcher? Can you open the SenseeAct app on the phone? What is this app saying? We can also have a short call today.

**Comment by aversluis1** [comment]
@bscheltinga, I indeed get the notification to contact the researcher. The senseeAct app is telling me 'we are measuring your PA and HR from your Garmin smartwatch'. Let me know if a call is handy, I am pretty available today until 14.

**Comment by bscheltinga** [comment]
@aversluis1 Is the name of the coach in Niceday 'Coach Sam Test'? If yes, could you try the following:
1. Run `/EXTERNAL_closing_dialog`, you will receive a few messages, at some point it will say: "In totaal heb je <number steps> (...)". 
2. Note down this printed number, then type: `/restart`
3. Take some steps and wait about 10 minutes. Make sure you have bluetooth and internet connections.
4. Run `/EXTERNAL_closing_dialog` again.
Did the number change? If yes, it would mean that your steps are now actually tracked. If No, let's have a call.

**Comment by aversluis1** [comment]
@bscheltinga, I ran it, but the number of steps unfortunately did not change.. Can you call me? 0655345684

**Comment by wbaccinelli** [comment]
@bscheltinga I see steps recorded today, but the previous recording is from 04-09-2023

**Comment by bscheltinga** [comment]
@wbaccinelli I just had a call with Anke, the steps recorded from today where uploaded after the call I expect. We tried a few different things before it worked again:
1. See what the senseeact phone app displayed (it said: measuring)
2. Activate the senseeact watch app (still not working)
4. Log out and log in into the senseeact app

After the 3rd step, data was coming in.

Interestingly, Anke did not have to relogin after doing the update of the senseeact smartphone app, while Milon and I had to. This is probably why no data was coming in.
